### PR TITLE
showImages: Align expando-box when exceeding width of body

### PR DESCRIPTION
**Pull request by larsjohnsen**
It will right-align if there's free space on the left side; if not, then left-align.

### CODE REVIEW

File: lib/modules/showImages.js [hunk 1]
```diff
@@ -1158,6 +1159,8 @@ function generateIframe(options) {
 		return removeInstead;
 	};
 
+	keepMediaVisible(iframeNode);
```
Comment (larsjohnsen) [inline lib/modules/showImages.js]: Note: This does nothing without #3027

**Comment by erikdesjardins** [comment]
This appears to break image moving (shift + drag) for images with captions (not completely sure if that's the cause), e.g. [this one](https://drscdn.500px.org/photo/93192709/m=2048/9b0c79faf3c0b3dc5663c75accd184cb). [Test comment](https://www.reddit.com/r/restests/comments/46ymr0/test_all_the_expandos/d3sb2kc) (subject to change).

**Comment by larsjohnsen** [comment]
Thanks, @erikdesjardins. Fixed now.

File: lib/modules/showImages.js [hunk 2]
```diff
@@ -1058,7 +1070,8 @@ function generateImage(options) {
 	options.openInNewWindow = module.options.openInNewWindow.value;
 
 	const element = $(imageTemplate(options))[0];
-	const image = element.querySelector('img');
+	const image = element.querySelector('img.res-image-media');
+	const anchor = element.querySelector('a.res-expando-link');
```
Comment (larsjohnsen) [inline lib/modules/showImages.js]: For some reason isn't this change included in any of the individual commit diffs.

**Comment by erikdesjardins** [comment]
Demo gif because why not:

**Pull request merged**